models/aeif_cond_alpha.nestml

```diff
@@ -74,11 +73,6 @@ neuron aeif_cond_alpha:
     tau_syn_ex ms = 0.2 ms    # Synaptic Time Constant Excitatory Synapse
     E_in mV = -85.0 mV        # Inhibitory reversal Potential
     tau_syn_in ms = 2.0 ms    # Synaptic Time Constant for Inhibitory Synapse
-
-    # Input current injected by CurrentEvent.
-    # This variable is used to transport the current applied into the
-    # _dynamics function computing the derivative of the state vector.
-    I_stim pA = 0 pA
```

Comment:
I think this removal is correct and in line with what I said about the other current inputs. The name and comment of this one suggests that this was actually meant to be what *currents* was made to do? Maybe *I_stim* is a good name for the current buffer in the `input` block?

Reply:
proposing here again ``external_current`` or ``current_input``